parts/linux/cloud-init/artifacts/components.json

```diff
@@ -650,8 +650,7 @@
             "versionsV2": [
               {
                 "renovateTag": "name=moby-containerd, os=ubuntu, release=20.04",
-                "latestVersion": "1.7.20"
-              }
+                "latestVersion": "1.7.22-ubuntu20.04u1"}
```

Comment:
nit: Is the closing bracket without indentation something that Renovate/we can edit?

Reply:
yaya, this is known. I will follow up with this. Thanks for catching.

Reply:
why does this patch say ubuntu 20.04?

Reply:
So I found that with `ubuntu 20.04`, it's actually the complete version of moby-containerd. We usually just put `1.7.20` without the patch and I guess it's apt-get that automatically resolve to the latest complete version with patch. But Renovate will update to the latest patch version so we also see it here. Let me know if this is not clear enough.